app/views/api/services/new.html.slim

```diff
@@ -1,6 +1,7 @@
 = javascript_pack_tag 'new_service'
 
 #new_service_wrapper data-new-service-data={ isServiceDiscoveryAccessible: service_discovery_accessible?,
+                                             template: @service.as_json,
```

Comment:
I'm not convince of the naming here. 'Template' doesn't sound right and it gets more confusing in React.
The form is for a _service_, as it is defined in the controller. Perhaps _new controller_ even, but template sounds out of place IMHO.

Turns out this object contains errors and it really is a `NewServiceTemplateProps`. So is it a template, a service or a props? I think it would make more sense for the frontend to pass both `service` and `errors` to the Form component as two different props. This way it'd more explicit and we'd be handling well-kown entities (a service and an array of error messages).

We would avoid also defining prop types such as `NewServiceTemplateProps` that belong at a component level. I think `ServiceFormTemplate` would work better as a general type (reusable not only for "new service" forms but also for "edit service" ones) and then `type NewServiceFormProps = { template: ServiceFormTemplate }`.

Hope it makes sense!

Reply:
You are right about the name of the types, I'll change accordingly.